narratives/ncov_sit-rep_it_2020-03-13.md

```diff
@@ -192,15 +192,15 @@ Ingrandendo la cartina, si può osservare che ci sono molte connessioni tra l'It
 # [Trasmissione locale nelle Isole britanniche e in Irlanda](https://nextstrain.org/ncov/2020-03-13?c=country&legend=closed&d=tree&f_country=United%20Kingdom,Ireland&p=full)
 
 <!-- Looking at the British Isles & Ireland as an example, we can see several instances where viruses that are closely related to samples from other countries appear in the British Isles & Ireland.-->
-Prendendo le Isole britanniche e l'Irlanda come esempio, osserviamo diversi casi in cui compaiono nelle Isole Britanniche e in Irlanda virus molto simili a quelli presenti in altri paesi.
+Prendendo le Isole Britanniche e l'Irlanda come esempio, osserviamo diversi casi in cui in questi luoghi compaiono virus molto simili a quelli presenti in altri paesi.
```

Comment:
Please use capital letter in "Paesi" when it means "countries".

Reply:
Hi @sarameiqi, fixed that and a capital letter in the title. Let me know if you spot further improvements.